routemaster/mixins/log.rb

```diff
@@ -66,6 +70,22 @@ def _smart_backtrace(e)
           matches || !seen_own
         end
       end
+
+      def _log_level_constant
+        if _log_level_invalid?
+          Logger::INFO
+        else
+          Logger.const_get(_fetch_log_level)
```

Comment:
`NameError` seemed too generic to catch to me as it might be raised in any case you misspelled the variable name or something that kind, so it we catch wouldn't it make debugging bit harder for this part of the code?